src/hotspot/share/cds/dumpTimeClassInfo.hpp

```diff
@@ -209,8 +209,15 @@ class DumpTimeSharedClassTable: public DumpTimeSharedClassTableBaseType
     }
```

Comment:
This file has a blank first line, can you fix this?

Reply:
And no new line between these here:
#define SHARED_CDS_DUMPTIMESHAREDCLASSINFO_HPP
#include "cds/archiveBuilder.hpp"

Reply:
Fixed.